Wireframe/css/style.css

```diff
@@ -0,0 +1,71 @@
+*{
+    text-align: center;
+    font-family: 'Courier New', Courier, monospace;
+    
+   
+}
+
+header{
+    background-color: rgba(255, 98, 0, 0.432);
+    color: white;
+    padding: 9px;
+    margin-bottom: 10px;
+}
+
+main{
+    display: grid;
+    gap:10px;
+    grid-template-columns: 1fr 1fr;
+    grid-template-rows: auto auto;
+}
+
+.one{
+    border: 2px solid #ddd;
+    grid-column: span 2;
+    padding: 20px;
+}
+
+
+
+.two{
+    border: 2px solid #ddd;
+    grid-column: span 1;
+    grid-row: span 1;
+    padding: 20px;
+}
+
+.three{
+    border: 2px solid #ddd;
+    grid-row: span 1;
+    grid-column: span 1;
+    padding: 20px;
+}
+
+
+footer{
+    background-color: rgba(255, 98, 0, 0.432);
+    color: white;
+    padding: 9px;
+    margin-top: 10px;
+}
+
+button{
+    background-color: rgb(255, 98, 0);
+    border: rgb(255, 98, 0) solid;
+    border-radius: 3px;
+    color:white;
+   
+}
+
+a{
+    color:black;
+    text-decoration: none;
+}
+
+a:hover{
+    cursor:pointer;
+}
+
+button:hover{
+    cursor:pointer;
+}
```

Comment:
As per responsive layout, let's make sure that the page looks nice on any device. In our case here, it should be enough to adjust the grid structure for a mobile viewport (which is usually ~620px and smaller).

Luckily we can use media queries!  Here is [some example ](https://www.w3schools.com/css/css_rwd_mediaqueries.asp#:~:text=Try%20it%20Yourself%20%C2%BB-,Another%20Breakpoint,-You%20can%20add)of using it!